discovery_engine_core/web-api/src/db.rs

```diff
@@ -69,21 +68,27 @@ pub(crate) fn init_db(config: &InitConfig) -> Result<Db, Box<dyn std::error::Err
     let documents = articles
         .into_iter()
         .map(|article| {
-            let id = article
+            let article_id = article
                 .get("id")
                 .expect("Article needs to have an 'id' field")
                 .as_str()
-                .expect("The 'id' field needs to be represented as String")
-                .try_into()
-                .expect("The 'id' field needs to be a valid UUID");
+                .expect("The article's 'id' field needs to be represented as String")
+                .trim()
+                .to_string();
+
+            assert!(
+                !article_id.is_empty(),
```

Comment:
🔧  The Stroy description also notes that the string must not contain zero bytes. We could check it with `article_id.contains('\u{0000}')`
same fro the UserID